modules/programs/rofi-pass.nix

```
@@ -0,0 +1,32 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.programs.rofi.pass;
+in {
+  meta.maintainers = [ maintainers.seylerius ];
+
+  options.programs.rofi.pass = {
+    enable = mkEnableOption "rofi integration with password-store";
+    stores = mkOption {
```

Separate options by an empty line.
```suggestion

    stores = mkOption {
```